bin/sh/install-boost.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+if [ ! -d "$BOOST_ROOT/lib" ]
+then
```

Comment:
Consider again extracting out the version number `1_59_0` so we can upgrade and change just in one place!  :-)